### PR TITLE
Update to 3.38.1

### PR DESCRIPTION
**Pull request by A6GibKm**

### CODE REVIEW

**Comment by A6GibKm** [comment]
bot, build org.gnome.Boxes

**Comment by A6GibKm** [comment]
The reason both this and #54 are marked as WIP is because whenever I exit the app I am not able to open the boxes again. Have in mind that this bug also affects me on the `stable` branch, so it is probably not related to the PRs.

**Comment by A6GibKm** [comment]
bot, build org.gnome.Boxes

**Comment by A6GibKm** [comment]
Tested both PRs on my laptop and they worked just fine :+1:. Seems boxes does not run very well on my other machine, even the stable branch.

**Comment by Eonfge** [comment]
@bilelmoussaoui All seems good to me

File: org.gnome.Boxes.json [hunk 1]
```diff
@@ -506,10 +480,9 @@
             ],
             "sources" : [
                 {
-                    "disable-shallow-clone" : true,
-                    "type" : "git",
-                    "url" : "https://gitlab.gnome.org/gnome/gnome-boxes.git",
-                    "branch" : "gnome-3-38"
+                    "type": "archive",
```
Comment (felipeborges) [inline org.gnome.Boxes.json:483]: I would like to continue pulling these changes from the stable gnome-3-38 branch, so I can perform post release fixes without making new releases.

**Comment by felipeborges** [comment]
Thanks for the PR! I appreciate it.

File: org.gnome.Boxes.json (hunk 1 above)
Comment (A6GibKm) [reply org.gnome.Boxes.json:483]: While that might be convenient, it usually breaks the install, if you like point me to a commit on the master branch and I will set it to it.

Comment (felipeborges) [reply org.gnome.Boxes.json:483]: we have been doing this forever.

Comment (felipeborges) [reply org.gnome.Boxes.json:483]: oh, and the gnome-3-38 branch, not master.

Comment (A6GibKm) [reply org.gnome.Boxes.json:483]: Ok, will set it to it, but it just broke for me when doing this PR. And it was also not possible to build the current manifest I think.

**Pull request merged**